snuba/datasets/replays_processor.py

```diff
@@ -133,7 +134,7 @@ def process_message(
         self, message: Mapping[Any, Any], metadata: KafkaMessageMetadata
     ) -> Optional[ProcessedMessage]:
         try:
-            replay_event = message["payload"]
+            replay_event = json.loads(bytes(message["payload"]))
```

Comment:
I would suggest you use rapidjson instead of json since it offers better performance. Most of our consumers use rapidjson

Reply:
oh yeah, was gonna ask about that. yeah totally makes sense.